examples/dao/test/dao-app-flow.js

```diff
@@ -944,5 +950,44 @@ describe("DAO test", function () {
 		]) {
 			assert.isUndefined(proposalALsigAcc.getLocalState(appID, key));
 		}
+
+		//verify that the funds from proposalLsig can only be withdrawed back to the owner
+		const withdrawTx = [
+			{
+				type: types.TransactionType.TransferAsset,
+				sign: types.SignType.LogicSignature,
+				fromAccountAddr: proposalALsigAcc.address,
+				toAccountAddr: proposerA.address,
+				amount: 1,
+				lsig: proposalALsig,
+				assetID: govTokenID,
+				payFlags: {},
+			},
+		];
+
+		let proposerABalance = proposerA.getAssetHolding(govTokenID).amount;
+		let proposalLsigABalance = proposalALsigAcc.getAssetHolding(govTokenID).amount;
+
+		runtime.executeTx(withdrawTx);
+		syncAccounts();
+
+		assert.equal(proposerA.getAssetHolding(govTokenID).amount, proposerABalance + 1n);
+		assert.equal(
+			proposalALsigAcc.getAssetHolding(govTokenID).amount,
+			proposalLsigABalance - 1n
+		);
+
+		//verify that the funds from proposalLsig cannot be withdrawed to an non-owner account
+		runtime.optIntoASA(govTokenID, alice.address, {});
+		[alice] = runtime.defaultAccounts();
```

Comment:
this line is not needed

Reply:
`[alice] = runtime.defaultAccounts();`
Is needed for the assertion below where we check if the account opted-in to the GOV token
`assert.isDefined(alice.getAssetHolding(govTokenID));`